CMakeLists.txt

```diff
@@ -16,9 +16,24 @@ project(deepstate)
 cmake_minimum_required(VERSION 2.8)
 
 option(DEEPSTATE_LIBFUZZER, OFF)
+if(DEFINED ENV{DEEPSTATE_LIBFUZZER})
+  set(DEEPSTATE_LIBFUZZER ON)
```

Comment:
Is this okay that setting the envvar to *any* variable will set the option?

As in:
```
➜  cmake-envvars cat test.cmake
IF(DEFINED ENV{DEEPSTATE_LIBFUZZER})
    MESSAGE(STATUS "DEEPSTATE_LIBFUZZER env seen: --[$ENV{DEEPSTATE_LIBFUZZER}]--")
ELSE()
    MESSAGE(STATUS "DEEPSTATE_LIBFUZZER is not set")
ENDIF()
➜  cmake-envvars cmake -P test.cmake
-- DEEPSTATE_LIBFUZZER is not set
➜  cmake-envvars DEEPSTATE_LIBFUZZER=1 cmake -P test.cmake
-- DEEPSTATE_LIBFUZZER env seen: --[1]--
➜  cmake-envvars DEEPSTATE_LIBFUZZER=0 cmake -P test.cmake
-- DEEPSTATE_LIBFUZZER env seen: --[0]--
➜  cmake-envvars DEEPSTATE_LIBFUZZER=off cmake -P test.cmake
-- DEEPSTATE_LIBFUZZER env seen: --[off]--
➜  cmake-envvars
```

Reply:
README:
`run cmake when you install with the BUILD_LIBFUZZER environment variable defined`. Thats why. IDK if its behaviour we want.

Also for Angora build, it uses: https://github.com/trailofbits/deepstate/blob/8cfd58c950b8640205eb5b1673e37947c8aba304/CMakeLists.txt#L174

Reply:
Hm, then I think we can accept those, but let's double check with someone more involved in the project.